src/test/java/systemtests/EntryBookSystemTest.java

```diff
@@ -173,19 +173,26 @@ protected void executeCommand(String command) {
     }
 
     /**
-     * Displays all entries in the address book.
+     * Displays all entries in the list entry book.
      */
-    protected void showAllEntries() {
+    protected void showAllListEntries() {
         executeCommand(ListCommand.COMMAND_WORD);
         assertEquals(getModel().getListEntryBook().getEntryList().size(), getModel().getFilteredEntryList().size());
     }
 
+    /**
+     * Displays all entries in the archives entry book.
+     */
+    protected void showAllArchivesEntries() {
+        executeCommand(ArchivesCommand.COMMAND_WORD);
+        assertEquals(getModel().getArchivesEntryBook().getEntryList().size(), getModel().getFilteredEntryList().size());
+    }
+
     /**
      * Displays all entries with any parts of their titles matching {@code keyword} (case-insensitive).
      */
     protected void showEntriesWithTitle(String keyword) {
         executeCommand(FindCommand.COMMAND_WORD + " " + PREFIX_TITLE + keyword);
-        assertTrue(getModel().getFilteredEntryList().size() < getModel().getListEntryBook().getEntryList().size());
```

Comment:
This line is removed because this is not true all the time, even back in AB4.

Reply:
![image](https://user-images.githubusercontent.com/829070/55744603-e87abd00-5a67-11e9-8945-0a34c515907c.png)